hal/network/ncp/cellular/cellular_ncp_client.h

```diff
@@ -81,6 +81,14 @@ enum class CellularAccessTechnology {
     LTE_NB_IOT = 9
 };
 
+enum class CellularOperationMode {
+    NONE = -1,
+    PS_ONLY = 0,
+    CS_PS_MODE = 2,
```

Comment:
Once a device switches to `PS_ONLY` mode, is there ever a reason to switch it back to `CS_PS_MODE`? 
Is there a reason we arent unconditionally switching everything to `PS_ONLY`? (Presumably because we arent fully sure of the behavior change yet?)

Reply:
No reason to switch back to CS_PS_MODE other than for testing purposes.  Other devices have been found to already be in PS_ONLY mode by default, so R510 only is to keep the change focused.